submissions/2025-05-22-OpenFF-Sage-2.1.0-Optimization-Training-Dataset-v1.0/ds_info.json

```diff
@@ -0,0 +1,7 @@
+{
+    "dataset_name": "OpenFF Sage 2.1.0 Optimization Training Dataset v1.0",
+    "dataset_tagline": "B3LYP-D3BJ/DZVP conformers for training OpenFF 2.1.0 Sage with improved coverage of sulfonamides and phosphates and improved fitting procedures.",
+    "description": "A quantum chemical (QC) dataset curated to train the OpenFF 2.0.0 Sage forcefield, with reparametrized Lennard-Jones (LJ) and valence parameters. This QC dataset with the OpenFF default level of theory, B3LYP-D3BJ/DZVP, is used to train Sage parameters. This is the complete optimization dataset includes calculations used for training OpenFF 2.0.0 Sage in addition to other, consisting of the following datasets: 'OpenFF Gen 2 Opt Set 1 Roche', 'OpenFF Gen 2 Opt Set 2 Coverage', 'OpenFF Gen 2 Opt Set 3 Pfizer Discrepancy', 'OpenFF Gen 2 Opt Set 4 eMolecules Discrepancy', 'OpenFF Gen 2 Opt Set 5 Bayer', 'OpenFF Gen2 Optimization Dataset Protomers v1.0', 'OpenFF Iodine Chemistry Optimization Dataset v1.0', 'OpenFF Optimization Set 1', 'SMIRNOFF Coverage Set 1', and 'OpenFF Aniline Para Opt v1.0'. These combined datasets were filtered with `ElementFilter(allowed_elements=['H', 'C', 'N', 'O', 'S', 'P', 'F', 'Cl', 'Br'])`, `RecordStatusFilter(status=RecordStatusEnum.complete)`, `ConnectivityFilter(tolerance=1.2)`, `UnperceivableStereoFilter()`, `ConformerRMSDFilter(max_conformers=12)`,and `ChargeCheckFilter()`.",
```

Comment:
Could you please update this to match the changes in the README description too? :-)